### DISCUSSION TITLE
TCAS issue ?

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#5064] [General]
Hi developers,

no Aircraft on ND are displayed. TCAS is active .... Sometimes I see some on the ND, but not often ...

![grafik](https://user-images.githubusercontent.com/76531893/121236314-1a7d4900-c896-11eb-8352-7e8a8c075fd9.png)

greets Klaus

**Comment by tracernz** [comment]
Could be https://github.com/flybywiresim/a32nx/pull/5059

**Reply by CptBayer** [reply]
hi,

thanx for the link... no, not really like described in 5059..... 

greets Klaus

**Reply by tracernz** [reply]
Give it a try after that one merges to master anyway. Otherwise, may be a sim bug.

**Reply by CptBayer** [reply]
I am using the dev version. Are there other people don´t get the other aircrafts displayed on the ND? Sometimes, but not often I can see some.... That´s what I am wondering for ....

greets Klaus

**Reply by CptBayer** [reply]
Ok.... I see planes when the range of the ND is 10 or 20 nm. Bigger than 20 nm no aircraft are displayed via TCAS ... Any suggestion ? ...As well no aircraft with a range of 10 or 20 nm when the selector is set to "All". When the selector is set to "Above", it is working again .... Selector on "Below" not checked out yet, cause was on ground ...  

greets Klaus

**Reply by tracernz** [reply]
> Ok.... I see planes when the range of the ND is 10 or 20 nm. Bigger than 20 nm no aircraft are displayed via TCAS ... Any suggestion ? ...As well no aircraft with a range of 10 or 20 nm when the selector is set to "All". When the selector is set to "Above", it is working again .... Selector on "Below" not checked out yet, cause was on ground ...
> 
> greets Klaus

It's an option on the real aircraft whether to show traffic on other ranges (and there's a warning message to switch range if there's an intruder and you're not in 10, 20 NM range if the option is not selected). Our implementation did not consider this option, so it only showed traffic on 10, 20 NM range. #5059 fixes this.

**Reply by 2hwk** [reply]
Note that multiplayer aircraft are not displayed by TCAS at this time (Coherent hook/asobo issue)

**Reply by tracernz** [reply]
> Note that multiplayer aircraft are not displayed by TCAS at this time (Coherent hook/asobo issue)

I thought they fixed that in sim update 4?

**Reply by CptBayer** [reply]
thanx tracernz,

I understand. Only the part about selcetor in "All" position might be an issue (read above)?

greets Klaus

**Reply by tracernz** [reply]
> thanx tracernz,
> 
> I understand. Only the part about selcetor in "All" position might be an issue (read above)?
> 
> greets Klaus

#5059 is now merged, so TCAS traffic should be visible on all ranges with the latest dev, as long as MSFS tells us about it.

**Reply by CptBayer** [reply]
yeah .... It is working right now. Saw it today :-) ...

greets Klaus

**Reply by tracernz** [reply]
Good to hear. Thanks for following up. 👍

**Reply by MStallen** [reply]
The same
